src/components/TokenInput.tsx

```diff
@@ -71,7 +71,7 @@ function TokenInput({
 
   return (
     <div {...rest}>
-      {max != null && (
+      {max && (
```

Comment:
wat

Reply:
it's safer to avoid coercion. is there a reason we need to coerce?

Reply:
I think this should be optional.